pkg/service/tap_service.go

```diff
@@ -45,65 +46,20 @@ func (tapService *TAPService) createTurboTargets(IsRegistered chan bool) {
 		return
 	}
 	glog.Infof("Probe " + tapService.ProbeCategory + "::" + tapService.ProbeType + " Registered : === Add Targets ===")
-	var targets []*probe.TurboTarget
-	targets = tapService.GetProbeTargets()
-	for _, targetInfo := range targets {
-		glog.Infof("Adding target %s", targetInfo)
-		tapService.AddTurboTarget(targetInfo)
-	}
-}
-
-// ==============================================================================
-
-// Configuration parameters for communicating with the Turbo server
-type TurboCommunicationConfig struct {
-	// Config for the Rest API client
-	*vmtapi.TurboAPIConfig
-	// Config for RemoteMediation client that communicates using websocket
-	*communication.ContainerConfig
-}
-
-func ParseTurboCommunicationConfig(configFile string) (*TurboCommunicationConfig, error) {
-	// load the config
-	turboCommConfig, err := readTurboCommunicationConfig(configFile)
-	if turboCommConfig == nil {
-		return nil, err
-	}
-	glog.Infof("WebscoketContainer Config : ", turboCommConfig.ContainerConfig)
-	glog.Infof("RestAPI Config: ", turboCommConfig.TurboAPIConfig)
-
-	// validate the config
-	// TODO: return validation errors
-	_, err = turboCommConfig.ValidateContainerConfig()
-	if err != nil {
-		return nil, err
+	targetInfos := tapService.GetProbeTargets()
+	for _, targetInfo := range targetInfos {
+		target := targetInfo.GetTargetInstance()
+		glog.V(4).Infof("Now adding target %v", target)
+		resp, err := tapService.AddTarget(target)
```

Comment:
at line 53, if the target already exist, it will return an error from API, something like 
```console
E0217 14:26:15.355142  124663 tap_service.go:55] Error during add target &{Custom Kubernetes my_k8s [0xc42065a840 0xc42065a880 0xc42065a8c0]}: Unsuccessful target addition response: {400 {"type":400,"exception":"com.vmturbo.api.exceptions.OperationFailedException: Failed to add target reason: Target already exists","message":"Failed to add target reason: Target already exists"} <nil>}
```